.phpcs.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<ruleset name="DD">
+	<description>Generally-applicable sniffs for Datadog Tracing</description>
+
+	<file>./</file>
+
+	<rule ref="PSR12" >
+	</rule>
+
+	<rule ref="Generic.Commenting.Todo"/>
+	<rule ref="Generic.Commenting.Fixme"/>
+    <rule ref="Generic.Commenting.DocComment.MissingShort">
```

Comment:
Maybe fix the indentation here?

Reply:
Yeah having consistent indentation sounds like a good idea 👍 😄